homeassistant/components/wemo/wemo_device.py

```diff
@@ -130,6 +132,15 @@ def async_update_listeners(self) -> None:
             update_callback()
 
 
+def _create_device_info(wemo: WeMoDevice) -> DeviceInfo:
```

Comment:
Let's merge this with the `_device_info` method. These are both pretty small methods that I think should just be one.

Reply:
I thought of that but isn't it easier to read if we make them separate?